app/javascript/packs/codemirror.ts

```diff
@@ -0,0 +1,8 @@
+import CodeMirror from 'codemirror'
+import 'codemirror/mode/htmlmixed/htmlmixed'
+import 'codemirror/lib/codemirror.css'
+import 'codemirror/theme/neat.css'
+
+import './codemirror-cms.scss'
```

Comment:
Remove this import and either rename the pack to `codemirror-cms` OR the scss to `codemirror`.